src/cli.rs

```diff
@@ -36,61 +46,62 @@ pub(crate) struct Args {
     pub render: Option<PathBuf>,
 
     /// Override the operating system
-    #[clap(
+    #[arg(
         short = 'p',
         long = "platform",
         action = ArgAction::Append,
-        possible_values = ["linux", "macos", "windows", "sunos", "osx", "android"],
+        value_parser = EnumValueParser::<PlatformType>::new(),
     )]
     pub platforms: Option<Vec<PlatformType>>,
 
     /// Override the language
-    #[clap(short = 'L', long = "language")]
+    #[arg(short = 'L', long = "language")]
     pub language: Option<String>,
 
     /// Update the local cache
-    #[clap(short = 'u', long = "update")]
+    #[arg(short = 'u', long = "update")]
     pub update: bool,
 
     /// If auto update is configured, disable it for this run
-    #[clap(long = "no-auto-update", requires = "command_or_file")]
+    #[arg(long = "no-auto-update", requires = "command_or_file")]
     pub no_auto_update: bool,
 
     /// Clear the local cache
-    #[clap(short = 'c', long = "clear-cache")]
+    #[arg(short = 'c', long = "clear-cache")]
     pub clear_cache: bool,
 
     /// Use a pager to page output
-    #[clap(long = "pager", requires = "command_or_file")]
+    #[arg(long = "pager", requires = "command_or_file")]
     pub pager: bool,
 
     /// Display the raw markdown instead of rendering it
-    #[clap(short = 'r', long = "--raw", requires = "command_or_file")]
+    #[arg(short = 'r', long = "raw", requires = "command_or_file")]
     pub raw: bool,
 
     /// Suppress informational messages
-    #[clap(short = 'q', long = "quiet")]
+    #[arg(short = 'q', long = "quiet")]
     pub quiet: bool,
 
     /// Show file and directory paths used by tealdeer
-    #[clap(long = "show-paths")]
+    #[arg(long = "show-paths")]
     pub show_paths: bool,
 
     /// Create a basic config
-    #[clap(long = "seed-config")]
+    #[arg(long = "seed-config")]
     pub seed_config: bool,
 
     /// Control whether to use color
-    #[clap(
+    #[arg(
         long = "color",
         value_name = "WHEN",
-        possible_values = ["always", "auto", "never"]
+        value_parser = EnumValueParser::<ColorOptions>::new(),
```

Comment:
We don't need to be so verbose here, we can use `#[arg(value_enum)]` or even not specify anything and clap will figure this out on its own (same for the other value enum)
```suggestion
```

Reply:
Oh, nice 🙂